qiskit_ibm_runtime/api/clients/runtime.py

```diff
@@ -278,3 +278,60 @@ def job_logs(self, job_id: str) -> str:
     def logout(self) -> None:
         """Clear authorization cache."""
         self.api.logout()
+
+    # IBM Cloud only functions
+
+    def list_backends(self, timeout: Optional[float] = None) -> List[Dict[str, Any]]:
```

Comment:
This parameter appears to make sense for most calls. Other calls seem to use the [default](https://github.com/Qiskit/qiskit-ibm-runtime/blob/f847c86841f8d1f69423e83d9d429508daece672/qiskit_ibm_runtime/api/session.py#L148) value set during session initialization. Why is it explicitly needed in this case?

Reply:
This API used to take long since all the backends and their configuration were fetched in one call. So the user wanted the ability to pass an increased timeout in case backends() call fails. Now that the configuration is fetch in separate calls, we should no longer need this timeout. Have removed it.